compiler/rustc_middle/src/arena.rs

```diff
@@ -89,7 +89,6 @@ macro_rules! arena_types {
             [] name_set: rustc_data_structures::unord::UnordSet<rustc_span::Symbol>,
             [] autodiff_item: rustc_ast::expand::autodiff_attrs::AutoDiffItem,
             [] ordered_name_set: rustc_data_structures::fx::FxIndexSet<rustc_span::Symbol>,
-            [] pats: rustc_middle::ty::PatternKind<'tcx>,
```

Comment:
I assume this was just never used?

Reply:
So, the `IntoKind` impl wants `PatternKind` to be Copy, so I added that derive, but then there are overlapping arena impls, so I just removed this.

Reply:
Oh, right, as soon as an impl is copy it's automatically arena internable.